src/WholeBodyStateDisplay.cpp

```diff
@@ -407,6 +435,26 @@ void WholeBodyStateDisplay::updateCoPColorAndAlpha() {
   context_->queueRender();
 }
 
+void WholeBodyStateDisplay::updateFootCoPEnable() {
+  foot_cop_enable_ = foot_cop_enable_property_->getBool();
+  use_contact_status_in_foot_cop_ = foot_cop_enable_status_property_->getBool();
+  if (foot_cop_visual_.size() != 0 && !foot_cop_enable_) {
+    foot_cop_visual_.clear();
+  }
+  context_->queueRender();
+}
+
+void WholeBodyStateDisplay::updateFootCoPColorAndAlpha() {
+  const float &radius = foot_cop_radius_property_->getFloat();
+  Ogre::ColourValue color = foot_cop_color_property_->getOgreColor();
+  color.a = foot_cop_alpha_property_->getFloat();
+  for (size_t i = 0; i < foot_cop_visual_.size(); ++i) {
+    foot_cop_visual_[i]->setColor(color.r, color.g, color.b, color.a);
+    foot_cop_visual_[i]->setRadius(radius);
+  }
+  context_->queueRender();
+}
+
```

Comment:
Minimally adjusted these methods from the equivalent methods for friction cones and ground reaction forces.